src/includes/capture-error/flutter.mdx

```diff
@@ -0,0 +1,23 @@
+You can capture any exception object that you caught:
+
+```dart
+import 'package:sentry/sentry.dart';
+
+try {
+  aMethodThatMightFail();
+} catch (exception, stackTrace) {
+  await Sentry.captureException(
+    exception,
+    stackTrace: stackTrace,
+  );
+}
+```
+
+## Tips for Catching Errors
+
+- Use a `try/catch` block
+- Use a `catchError` block for `Futures`
+- The SDK already runs your init `callback` on an error handler, such as using `runZonedGuarded`, are captured automatically
+- Flutter-specific errors, such as using `FlutterError.onError`, are captured automatically
```

Comment:
this is the only line that changes from the dart.mdx to flutter.mdx, would be possible to special case and use a single file maybe?

Reply:
I'll check if I can nest the Dart include into the Flutter include.